specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2023-08-01/vaults.json

```diff
@@ -0,0 +1,2233 @@
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next chunk of the response",
+          "type": "string"
+        }
+      }
+    },
+    "ClientDiscoveryValueForSingleApi": {
+      "description": "Available operation details.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the Operation.",
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/ClientDiscoveryDisplay",
+          "description": "Contains the localized display information for this particular operation"
+        },
+        "origin": {
+          "description": "The intended executor of the operation;governs the display of the operation in the RBAC UX and the audit logs UX",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/ClientDiscoveryForProperties",
+          "description": "ShoeBox properties for the given operation."
+        }
+      }
+    },
+    "CrossSubscriptionRestoreSettings": {
+      "description": "Settings for Cross Subscription Restore Settings",
+      "type": "object",
+      "properties": {
+        "crossSubscriptionRestoreState": {
+          "enum": [
+            "Enabled",
+            "Disabled",
+            "PermanentlyDisabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "crossSubscriptionRestoreState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "RestoreSettings": {
+      "description": "Restore Settings  of the vault",
+      "type": "object",
+      "properties": {
+        "crossSubscriptionRestoreSettings": {
+          "description": "Settings for CrossSubscriptionRestore",
+          "$ref": "#/definitions/CrossSubscriptionRestoreSettings"
+        }
+      }
+    },
+    "ImmutabilitySettings": {
+      "description": "Immutability Settings of vault",
+      "type": "object",
+      "properties": {
+        "state": {
+          "enum": [
+            "Disabled",
+            "Unlocked",
+            "Locked"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ImmutabilityState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "SoftDeleteSettings": {
+      "description": "Soft delete Settings of vault",
+      "type": "object",
+      "properties": {
+        "softDeleteState": {
+          "enum": [
+            "Invalid",
+            "Enabled",
+            "Disabled",
+            "AlwaysON"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SoftDeleteState",
+            "modelAsString": true
+          }
+        },
+        "softDeleteRetentionPeriodInDays": {
+          "format": "int32",
+          "description": "Soft delete retention period in days",
+          "type": "integer"
+        },
+        "enhancedSecurityState": {
```

Comment:
>enhancedSecurityState [](http://example.com/codeflow?start=9&length=21)

I dont see an example json with this new enum, am I missing anything ?

Reply:
Added in example